src/open_inwoner/js/components/accessibility/enlarge_font.js

```diff
@@ -1,29 +1,95 @@
-class EnlageFont {
+export class EnlargeFont {
+  static selector = '.accessibility--enlarge-font'
+
   constructor(node) {
     this.node = node
     this.text = node.querySelector('.link__text')
     this.icon = node.querySelector('.material-icons')
     this.node.addEventListener('click', this.enlarge.bind(this))
+
+    this.root = document.documentElement
+
+    // Target specific sizes that need to be enlarged
+    this.styles = {
+      bodyFontSizeDefault: '--utrecht-document-font-size',
+      paragraphFontSizeDefault: '--utrecht-paragraph-font-size',
+      paragraphFontSizeSmall: '--utrecht-paragraph-small-font-size',
+      headingThreeFontSize: '--utrecht-heading-3-font-size',
+      headingFourFontSize: '--utrecht-heading-4-font-size',
+      // Legacy styles' initial values
+      oipBodyFontSizeDefault: '--font-size-body',
+      oipBodyFontSizeSmall: '--font-size-body-small',
+      oipHeadingThreeFontSize: '--font-size-heading-3',
+      oipHeadingFourFontSize: '--font-size-heading-4',
+      // OIP specific card-heading
+      oipCardHeadingFontSize: '--font-size-heading-card',
+    }
+
+    // Set initial values for toggling default styles
+    this.baseSizes = {
+      default: '16px',
+      small: '14px',
+      headingThree: '20px',
+      headingFour: '16px',
+      cardHeading: '18px',
+    }
+
+    // Enlarge the different types of body-font as well as paragraphs
+    // and set all lower types of headings (lower than H2) to get the same larger font-size
+    this.enlargeSizes = {
+      default: '20px',
+      small: '17px',
```

Comment:
The ratio between default and small should be constant (unless there's a reason to depart; it may well be desirable to have a slightliy different ration in the case of larger fonts): in `this.baseSizes` it is `16 - 14 = 2`, here we have `20 - 17 = 3`.

Reply:
If you'd set Small to 18px when enlarged, I personally think it looks a bit ugly, and I do not really see a clear difference with the default font-size anymore. Where I feel like you can really visually see a difference between small and default in the original sizes. 
Visually it doesn't seem to be an 'absolute number' thing but more of an 'optical' thing, but I can change this if it would really be an improvement, even though the font-size actually falls outside of the scope of the issue - but currently the toggling in the accessibility-header doesn't to work at all so I wanted to add an extra improvement.

Reply:
No, this is enough explanation. Thought it might be an accident.